official/recommendation/run.sh

```diff
@@ -58,7 +58,7 @@ do
   # To reduce variation set the seed flag:
   #   --seed ${i}
 
-  python -u "${SCRIPT_DIR}/ncf_main.py" \
+  python -u "${SCRIPT_DIR}/ncf_estimator_main.py" \
```

Comment:
It might make sense to have a thin shim run_keras.sh that for the most part uses run.sh, (similar to how run_tpu.sh works) That way it's very easy to reproduce issues.

Reply:
Done.